### PR TITLE
Homework3: Dmitry Makarevich

### PR DESCRIPTION
**Pull request by DzmitryMak**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle).
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks.
- [ ] my PR made from the fork feature branch to appropriate branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework3/task1.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""
+FizzBuzz
+Напишите программу, которая печатает цифры от 1 до 100,
+но вместо чисел, кратных 3 пишет Fizz, вместо чисел кратный 5
+пишет Buzz, а вместо чисел одновременно кратных и 3 и 5 - FizzBuzz
+"""
+
+for num in range(1, 101):
+    if not num % 3 and not num % 5:
```
Comment (belovmd) [inline src/homework3/task1.py]: 15?

File: src/homework3/task2.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+"""List practice
+1. Используйте генератор списков чтобы получить
+ следующий: ['ab', 'ac', 'ad', 'bb', 'bc', 'bd'].
+"""
+st = 'abcd'
+
+
+lst = [st[ind1] + st[ind2] for ind1 in range(2) for ind2 in range(1, 4)]
```
Comment (belovmd) [inline src/homework3/task2.py]: `for element in 'abcd'`?

File: src/homework3/task2.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ следующий: ['ab', 'ad', 'bc']
+"""
+print(lst[::2])
+
+
+"""
+3. Используйте генератор списков чтобы
+ получить следующий ['1a', '2a', '3a', '4a']
+"""
+lst_num = [str(ind_1) + 'a' for ind_1 in range(1, 5)]
```
Comment (belovmd) [inline src/homework3/task2.py]: '1234'?

File: src/homework3/task5.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+"""Уникальные элементы в списке
+Дан список. Выведите те его элементы, которые встречаются в списке
+только один раз. Элементы нужно выводить в том порядке,
+в котором они встречаются в списке.
+"""
+lst = [1, 31, 8, 2, 3, 32, 3, 5, 2, 22, 1, 8, 111]
+
+
+for i in range(0, len(lst)):
```
Comment (belovmd) [inline src/homework3/task5.py]: for element in lst?

**Review by belovmd — CHANGES_REQUESTED** [review]

File: src/homework3/task1.py (hunk 1 above)
Comment (DzmitryMak) [reply src/homework3/task1.py]: fixed

File: src/homework3/task2.py (hunk 2 above)
Comment (DzmitryMak) [reply src/homework3/task2.py]: fixed

File: src/homework3/task2.py (hunk 3 above)
Comment (DzmitryMak) [reply src/homework3/task2.py]: fixed

File: src/homework3/task5.py (hunk 4 above)
Comment (DzmitryMak) [reply src/homework3/task5.py]: fixed

**Review by belovmd — APPROVED** [review]
9

**Pull request closed without merging**